tests/0081-admin.c

```diff
@@ -935,6 +935,7 @@ static void do_test_IncrementalAlterConfigs(rd_kafka_t *rk,
         /** Test the test helper, for use in other tests. */
         do {
                 const char *broker_id = tsprintf("%d", avail_brokers[0]);
+                const char *group_id  = "my-group";
```

Comment:
Let's use the first topic as group here as in different places. It's randomly generated and avoid problem with repeated tests.
```suggestion
                const char *group_id  = topics[0];
```